pkg/config/config_template.yaml

```diff
@@ -1354,7 +1354,7 @@ api_key:
   ## @param peer_tags - list of strings - optional
   ## @env DD_APM_PEER_TAGS - list of strings - optional
   ## [BETA] Optional list of supplementary peer tags that go beyond the defaults. The Datadog backend validates all tags
-  ## and will drop ones that are unapproved.
+  ## and will drop ones that are unapproved.Ï
```

Comment:
```suggestion
  ## and will drop ones that are unapproved.
```

Reply:
Not sure if this was placed here intentionally, let me know if there's some context around this.